ios-base/scripts/bootstrap.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/zsh
+
+GIT_REPO=$(git rev-parse --show-toplevel)
+GIT_DIR=$(git rev-parse --git-dir)
+
+# Install Homebrew
+if ! type "brew" > /dev/null; then
+    echo '`brew` not found. Install Homebrew'
+    ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+fi
+
+# rbenv
+if ! type "rbenv" > /dev/null; then
+    echo '`rbenv` not found. Install rbenv'
+    brew install rbenv rbenv-communal-gems
+
+    cat << EOS >> ~/.zshrc
+    if which rbenv > /dev/null; then eval "\$(rbenv init -)"; fi
+    export PATH="\$HOME/.rbenv/bin:\$PATH"
+    eval "\$(rbenv init -)"
+EOS
+fi
+
+source ~/.zshrc
```

Comment:
Hmm... This script supports only `zsh`. We should support other shells.
But I don't have any idea... 😢

Reply:
I think iOS developer should use `zsh` because it has become default shell in macOS now.

If you insist to use bash, my solution is using `$SHELL`.

```sh
if [ "$SHELL" = '/bin/bash' ]; then
    source ~/.bashrc
elif [ "$SHELL" = '/bin/zsh' ]; then
    source ~/.zshrc
else
    echo "Unsupported shell: ${SHELL}"
    exit 1
fi
```

Reply:
I see. OK. Please add this script? And then I will merge this 🙏

Reply:
Fixed in 52c5dfea